meson.build

```diff
@@ -33,6 +46,11 @@ if tmpfilesdir == '' or not fs.exists('/run/.containerenv')
   endif
 endif
 
+zshcompletionsdir = get_option('zsh_completions_dir')
+if zshcompletionsdir == ''
+  zshcompletionsdir = join_paths(get_option('datadir'), 'zsh', 'site-functions')
+endif
+
```

Comment:
Could you group this together with the other completion related lines?

Reply:
I *think* I was keeping it in the same order as they are in `meson_options.txt`.  I can change the grouping in both places.

Reply:
Done.